pkg/agent/cniserver/pod_configuration.go

```diff
@@ -445,10 +445,6 @@ func (pc *podConfigurator) reconcile(pods []corev1.Pod, containerAccess *contain
 	var podWg sync.WaitGroup
 
 	for _, pod := range pods {
-		// Skip Pods for which we are not in charge of the networking.
-		if pod.Spec.HostNetwork {
```

Comment:
Does this change work as expected on Linux?

Reply:
Yes, this is still needed on Linux, it is added in the Pod list options as alternative.